pkg/storage/fs/fs_test.go

```diff
@@ -16,7 +17,7 @@ func (d *FsTests) TestLocationFuncs() {
 
 func (d *FsTests) TestGetSaveListRoundTrip() {
 	r := d.Require()
-	r.NoError(d.storage.Save(module, version, mod, zip, info))
+	r.NoError(d.storage.Save(context.Background(), module, version, mod, zip, info))
```

Comment:
nit for later: you can create the context in a `BeforeTest` method on the `FsTests` struct